ui_framework/doc_site/src/views/expression/expression.html

```diff
@@ -0,0 +1,32 @@
+<div class="kuiExpression">
+  <div class="kuiExpression__item">
+    <button data-id="toggleExpressionButton" class="kuiExpression__button">
+      <span class="kuiExpression__buttonDescription">WHEN</span>
+      <span class="kuiExpression_buttonValue">count()</span>
+    </button>
+    <div  data-id="toggleExpressionPopover" class="kuiExpression__popover kuiExpression__popover--hidden">
+      <div class="kuiExpression__popoverTitle">
+        When
+      </div>
+      <div class="kuiExpression__popoverContent">
+        <select class="kuiSelect"><option label="count()" selected="selected">count()</option><option label="average()">average()</option><option label="sum()">sum()</option><option label="median()">median()</option><option label="min()">min()</option><option label="max()">max()</option></select>
+      </div>
+    </div>
+  </div>
+  <div class="kuiExpression__item">
+    <button data-id="toggleExpressionButton" class="kuiExpression__button">
+      <span class="kuiExpression__buttonDescription">IS ABOVE</span>
+      <span class="kuiExpression_buttonValue">100</span>
+    </button>
+    <div  data-id="toggleExpressionPopover" class="kuiExpression__popover kuiExpression__popover--rightAligned kuiExpression__popover--hidden">
```

Comment:
Looks like an extra space after `<div` here. And can you split the attributes across multiple lines?